Added a Postgres connection

blurges opened the pull request:
I added in a connection to a local postgres installation.

apmasell commented on frontend/src/App.js:
```diff
@@ -1,12 +1,37 @@
+	}
+	componentWillMount() {
+		const _this = this
+		axios.get('http://localhost:5035/api/developers')
```
Where is this URL coming from?

blurges replied:
The commit was missing a few files, sorry. This URL is from `routes.js` line 16.

blurges commented:
Re: `0a0060`

I changed the `Volunteers` table to include fields for a few more things I found in the CSV snippet I got from you and the printed form I got from Susan: 
- address, 
- interested in ad hoc activities, 
- willing to train, 
- completed accessibility training, 
- wants to receive Strand News, 
- membership number and membership expiry, 
- motivation and goals, 
- how did you hear about us, 
- additional info. 
I also added a boolean field to `Areas` to track if an area allows self-registration.

I did not include fields for the following info in `Volunteers`: 
- areas and awards info (e.g. last worked in library, 10 year pin awarded) - this will come from the `Areas` and `Awards` tables
- preferred start date and weekly availability - this will likely go into another table, `Avalability` or similar, when we get to schedules

The application form questions "Consent to share email within museum and volunteer administration" and "Consent to share email with other volunteers" are represented by one field, `nonAdminsCanView`.
